aemedge/blocks/cards/cards-factoid.css

```diff
@@ -0,0 +1,37 @@
+/* Factoid card styling - creates a centered dark blue box with white text */
+
```

Comment:
Please align the css with the definitions in slack:

https://adobe-dx-support.slack.com/archives/C06PK80B5A5/p1750794951061859

(nesting, media queries inmediatly below the mobile-first selector)

Reply:
i checked most of them.

@ndedios want me to apply spacing and CSS selector nesting too?
or would such changes be defer until performed on whole project to avoid inconsistencies?

cc @kunwarsaluja